test/integration/formservice/formservice.js

```diff
@@ -95,4 +97,142 @@ context('Formservice', function() {
       .wait('@routeFormPatientFields')
       .wait('@routeFormResponse');
   });
+
+  specify('display action form with a response', function() {
+    cy
+      .routeFormByAction()
+      .routeFormActionDefinition()
+      .routeFormActionFields()
+      .routeAction()
+      .routeLatestFormResponse()
+      .visit('/formapp/pdf/action/1', { noWait: true, isRoot: true });
+
+    cy
+      .get('iframe')
+      .should('have.attr', 'src', '/formservice/1');
+
+    cy
+      .window()
+      .then(win => {
+        win.postMessage({ message: 'form:pdf', args: {
+          definition: {
+            components: [
+              {
+                key: 'patient.fields.insurance',
+                type: 'container',
+                input: true,
+                label: 'Insurance',
+                tableView: false,
+                components: [
+                  {
+                    key: 'name',
+                    type: 'textfield',
+                    input: true,
+                    label: 'Insurance Name',
+                    tableView: true,
+                  },
+                ],
+              },
+            ],
+          },
+          formData: {
+            patient: {
+              fields: {
+                insurance: {
+                  name: 'Test Insurance Name',
+                },
+              },
+            },
+          },
+          formSubmission: {
+            patient: {
+              fields: {
+                insurance: {
+                  name: 'Test Insurance Name',
+                },
+              },
+            },
+          },
+          contextScripts: {},
+          reducers: [],
+        } }, win.origin);
+      });
+
+    cy
+      .get('[name="data[patient.fields.insurance][name]"]')
+      .should('have.value', 'Test Insurance Name');
+  });
+
+  specify('action formservice latest response from action tags', function() {
+    cy
+      .routeFormByAction(_.identity, '77777')
+      .routeFormActionDefinition()
+      .routeFormActionFields()
+      .routeAction(fx => {
+        fx.data.id = '1';
+        fx.data.relationships.form.data = { id: '77777' };
+        fx.data.relationships['program-action'] = { data: { id: '11111' } };
+        fx.data.relationships.flow = { data: { id: '1' } };
+        fx.data.relationships.patient = { data: { id: '1' } };
+
+        fx.data.attributes.tags = ['prefill-latest-response'];
+
+        return fx;
+      })
+      .routeLatestFormResponse()
+      .visit('/formapp/pdf/action/1', { noWait: true, isRoot: true });
+
+    cy
+      .wait('@routeLatestFormResponse')
+      .itsUrl()
+      .its('search')
+      .should('contain', 'filter[action.tags]=foo-tag')
+      .should('contain', 'filter[flow]=1')
+      .should('contain', 'filter[patient]=1');
```

Comment:
Are we okay just checking the URL filter for this one?

In `display action form with a response`, we already test that the latest response displays in the form.